gradle/gradle-publish.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+
+ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID') ?: ''
+ext["signing.password"] = System.getenv('SIGNING_KEY_PASSWORD') ?: ''
+ext["signing.secretKeyRingFile"] = System.getenv('SIGNING_KEY_FILE') ?: ''
+ext["SONATYPE_NEXUS_USERNAME"] = System.getenv('SONATYPE_NEXUS_USERNAME') ?: ''
+ext["SONATYPE_NEXUS_PASSWORD"] = System.getenv('SONATYPE_NEXUS_PASSWORD') ?: ''
+
+
+afterEvaluate {
+    android.libraryVariants.each { variant ->
+        def variantName = variant.name
+        def resolvedVersion = project.version
```

Comment:
Do these variables need to be defined separately or can we just access `variant.name` and `project.version` directly?